build.gradle

```diff
@@ -56,6 +56,8 @@ springBoot {
     mainClass = 'org.ow2.proactive.scheduling.api.Application'
 }
 
+project.ext.set("local", "")
```

Comment:
This change looks dangerous to me.

Are you intending to force using -Plocal ?